release/tasks/disk-uploader/README.md

```diff
@@ -12,7 +12,7 @@ When user runs [KubeVirt Tekton Tasks](https://github.com/kubevirt/kubevirt-tekt
 
 ### Parameters
 
-- **EXPORT_SOURCE_KIND**: The name of the export source kind
+- **EXPORT_SOURCE_KIND**: Specify the export source kind (vm, vmsnapshot, pvc)
```

Comment:
```suggestion
- **EXPORT_SOURCE_KIND**: Specify the export source kind (VM, VMsnapshot, PVC)
```
Please, use capitals to indicate that they are acronyms.

Reply:
I'm not sure about this. `vm`, `vmsnapshot` or `pvc` is the expected value. It won't work if the user will set another value than [1].

[1] https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/modules/disk-uploader/pkg/vmexport/vmexport.go#L21-L25

Reply:
Ah I see, so those are the expected values of the field. Maybe you can clarify it, at least confused me.